docgen/src/examples/e-commerce-infinite/App.js

```diff
@@ -68,43 +68,39 @@ const Facets = () => (
 
     <section className="facet-wrapper">
       <div className="facet-category-title facet">Show results for</div>
+
```

Comment:
nit: whitespace is not necessary here

Reply:
My bad, I don't like when the code is too compressed 🙂